projects/qt5.cmake

```diff
@@ -11,19 +11,19 @@
 # path (e.g. C:/Program Files/ovsrpro 0.0.1-vc120-64/qt5)
 ########################################
 xpProOption(qt5)
-set(QT5_VER v5.5.1)
-set(QT5_REPO http://code.qt.io/qt/qt5.git)
+set(QT5_VER 5.5.1)
+set(QT5_REPO http://code.qt.io/cgit/qt/qt5.git)
```

Comment:
I'm not sure what it stands for, but it's part of the path for the repo - the original url that was in there doesn't even exist.  I'm not sure if they changed it recently or what...